File: INICIANTE/1070/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int x = scanner.nextInt();
        x = x + 1 - x % 2;
        for (int odd = x; odd < x + 11; odd += 2)
            System.out.println(odd);
        scanner.close();
    }
}

File: INICIANTE/1070/main.c
#include <stdio.h>

int main() {
    int x;
    scanf("%d", &x);
    x = x + 1 - x % 2;
    for (int odd = x; odd < x + 11; odd += 2)
        printf("%d\n", odd);
    return 0;
}
